android/sample/src/main/java/com/boswelja/ephemeris/sample/compose/ComposeCalendarFragment.kt

```diff
@@ -42,15 +50,33 @@ class ComposeCalendarFragment : Fragment() {
 }
 
 @Composable
-fun CalendarScreen() {
+fun CalendarScreen(
+    headerDateFormatter: DateTimeFormatter = DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM)
+) {
     val calendarState = rememberCalendarState {
         CalendarMonthPageSource(
             DayOfWeek.SUNDAY
         )
     }
+
+    var headerText by remember { mutableStateOf("") }
```

Comment:
I heard you liked scope creep, so what if I suggested something like an `onDateChanged` to update this state instead of just a mutable remember state? :P

Reply:
As part of `EphemerisCalendar`? That would have to be a `ClosedRange<LocalDate>`, I like the idea but it does also deviate from the core CalendarState interface

Reply:
If you'd like to open an issue we can look into this in more depth